File: mgyoutube-multitech/api-go/src/app/datastoresproperties.go
package main

import (
	"encoding/json"
	"io/ioutil"
)

// DatastoresProperties - Datastores properties read from a config file
type DatastoresProperties struct {
	MongoConnectionString string `json:"mongoConnectionString"`
	MongoDatabaseName     string `json:"mongoDatabaseName"`
	CouchConnectionString string `json:"couchConnectionString"`
	CouchUsername         string `json:"couchUsername"`
	CouchPassword         string `json:"couchPassword"`
}

// ReadDatastoresProperties - read the config file and return the YouTube properties
func ReadDatastoresProperties() (*DatastoresProperties, error) {

	properties := DatastoresProperties{}

	file, errReadFile := ioutil.ReadFile("secrets/datastores.json")
	if errReadFile != nil {
		return nil, errReadFile
	}

	errUnmarshal := json.Unmarshal([]byte(file), &properties)

	if errUnmarshal != nil {
		return nil, errUnmarshal
	}

	return &properties, nil
}

File: mgyoutube-multitech/api-node/src/datastoresproperties.ts
import { readFileSync } from "fs"

export class DatastoresProperties {
    public couchConnectionString: string;
    public couchUsername: string;
    public couchPassword: string;
    public mongoConnectionString: string;
    public mongoDatabaseName: string;

    constructor() {
        var json = readFileSync("secrets/datastores.json");
        var jsonObject = JSON.parse(json.toString());

        this.couchConnectionString = jsonObject.couchConnectionString;
        this.couchUsername = jsonObject.couchUsername;
        this.couchPassword = jsonObject.couchPassword;
        this.mongoConnectionString = jsonObject.mongoConnectionString;
        this.mongoDatabaseName = jsonObject.mongoDatabaseName;
    }
}